cdctest: unskip sql smith for nemeses

wenyihu6 opened the pull request:
todo 

Epic: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/139509)
<!-- Reviewable:end -->

wenyihu6 committed:
debug

wenyihu6 committed:
kvfeed: fix span frontier minimum timestamp

When calculating the minimum timestamp of spans in the frontier, we encounter a
similar issue to #123371. Specifically, if the minimal timestamp tracked so far
is empty, we always write the span's timestamp.

Release note: todo (check on the impact)
Epic: none

wenyihu6 committed:
initial

wenyihu6 committed:
more decoding

wenyihu6 commented:
Fixed by https://github.com/cockroachdb/cockroach/pull/139914.
